README.md

```diff
@@ -91,7 +91,7 @@ My name is Marina, and my inspiration - Vanya The Cat :D. Glad to join the commu
 You can find me on [linkedin](https://www.linkedin.com/in/dm-poliuha/), or on [github](https://github.com/DmitryPoliuha)
 
 ![dima](gif/giphy-hi.gif)
- 
+
```

Comment:
Look, it will be easier for you to make changes only at the end of the file, because there are a lot of students editing it and you'll have merge conflicts.

Reply:
Thanks, you are right! I guess my default text editor for *.md files automatically made some changes. I need to investigate this issue.

Reply:
Yeap, you can configure it or just manually check before push changes